flycheck.el

```diff
@@ -8324,6 +8325,33 @@ See URL `http://www.scalastyle.org'."
                        ((not config-file) '(bold error))
                        (t 'success)))))))
 
+(flycheck-define-checker scheme-chicken
+  "A CHICKEN Scheme syntax checker using the CHICKEN compiler `csc'.
+
+See URL `http://call-cc.org/'."
+  :command ("csc" "-A" "-local" source)
```

Comment:
Can you explain why `-local` is needed here?
Also (but this is personal taste), I think `-analyze-only` reads better than `-A`.